Oracle/med_admin.sql

```diff
@@ -49,13 +49,29 @@ PMN_DROPSQL('drop index med_admin_idx');
 
 execute immediate 'truncate table med_admin';
 
-insert into med_admin(patid, encounterid, medadmin_providerid, medadmin_start_date, medadmin_start_time, medadmin_stop_date,
-medadmin_stop_time, medadmin_type, medadmin_code, medadmin_dose_admin, medadmin_dose_admin_unit, medadmin_source, raw_medadmin_med_name,
-raw_medadmin_code, raw_medadmin_dose_admin, raw_medadmin_dose_admin_unit, raw_medadmin_route)
-
+insert into med_admin(patid
+  , encounterid
+  , prescribingid
+  , medadmin_providerid
+  , medadmin_start_date
+  , medadmin_start_time
+  , medadmin_stop_date
+  , medadmin_stop_time
+  , medadmin_type
+  , medadmin_code
+  , medadmin_dose_admin
+  , medadmin_dose_admin_unit
+  , medadmin_route
+  , medadmin_source
+  , raw_medadmin_med_name
+  , raw_medadmin_code
+  , raw_medadmin_dose_admin
+  , raw_medadmin_dose_admin_unit
+  , raw_medadmin_route)
 with med_start as (
     select patient_num, encounter_num, provider_id, start_date, end_date, concept_cd, modifier_cd, instance_num
-    from BLUEHERONDATA.observation_fact
+    from &&i2b2_data_schema.observation_fact f
+    join encounter enc on enc.patid = f.patient_num and enc.encounterid = f.encounter_num
```

Comment:
This join killed med_admin performance, from 1 hour to incomplete after 18 hours.  I'll try to refactor this before leaving.  Worst case, I'll rollback the code and create a must do ticket for the next CDM build.